src/main/java/com/sap/sgs/phosphor/fosstars/tool/format/PrettyPrinter.java

```diff
@@ -99,22 +115,25 @@
     FEATURE_TO_NAME.put(OssFeatures.USES_OWASP_ESAPI, "Does it use OWASP ESAPI?");
     FEATURE_TO_NAME.put(OssFeatures.USES_OWASP_JAVA_ENCODER, "Does it use OWASP Java Encoder?");
     FEATURE_TO_NAME.put(OssFeatures.USES_OWASP_JAVA_HTML_SANITIZER,
-        "Does it use OWASP Java HTML Sanitizer??");
+        "Does it use OWASP Java HTML Sanitizer?");
     FEATURE_TO_NAME.put(OssFeatures.USES_DEPENDABOT, "Does it use Dependabot?");
-    FEATURE_TO_NAME.put(OssFeatures.USES_LGTM_CHECKS, "Does it use LGTM?");
+    FEATURE_TO_NAME.put(OssFeatures.USES_LGTM_CHECKS, "Does it use LGTM checks?");
     FEATURE_TO_NAME.put(OssFeatures.HAS_BUG_BOUNTY_PROGRAM, "Does it have a bug bounty program?");
     FEATURE_TO_NAME.put(OssFeatures.SIGNS_ARTIFACTS, "Does it sign artifacts?");
+    FEATURE_TO_NAME.put(OssFeatures.WORST_LGTM_GRADE, "The worst LGTM grade of the project");
+    FEATURE_TO_NAME.put(OssFeatures.FUZZED_IN_OSS_FUZZ, "Is it included to OSS-Fuzz?");
   }
 
   @Override
   public String print(RatingValue ratingValue) {
     StringBuilder sb = new StringBuilder();
     sb.append(String.format("Here is how the rating was calculated:%n"));
     sb.append(print(ratingValue.scoreValue(), INDENT_STEP, true, new HashSet<>()));
-    sb.append(String.format("Rating: %2.2f out of %2.2f -> %s%n",
-        ratingValue.score(), Score.MAX, ratingValue.label()));
-    sb.append(String.format("Confidence: %2.2f out of %2.2f%n",
-        ratingValue.confidence(), Confidence.MAX));
+    sb.append(String.format("Rating:     %s -> %s%n",
```

Comment:
There is a tab space in between. Is this a typo? or required?

Reply:
There is no tab, it's just spaces. Yes, I added it intentionally to align the numbers in Rating and Confidence fields.